# This is sent to ensure (in software) that only the nodes that are close to the drone will receive the message
geometry_msgs/Vector3 position

int32 drones_at_goal
